plug.vim

```diff
@@ -744,13 +745,17 @@ function! s:update_impl(pull, force, args) abort
   call append(0, ['', ''])
   normal! 2G
 
-  if s:ruby && s:update.threads > 1
+  if (s:py2|| s:ruby) && s:update.threads > 1
```

Comment:
Ah, one thing I forgot to mention.
Could you change this condition as follows, so that Neovim still prefers its own asynchronous installer?
```vim
if (s:py2 || s:ruby) && !s:nvim && s:update.threads > 1
```